grants_tagger/predict.py

```diff
@@ -15,6 +15,43 @@
 from typing import List, Optional
 
 
+def format_predictions(
+    Y_pred_proba, label_binarizer, threshold=0.5, probabilities=True
+):
+    """
+    Formats predictions to output a list of dictionaries
+
+    Y_pred_proba: sparse array or list of predicted probabilites or class
+    (i.e. the output of  `.predict` or `.predict_proba` classifier)
+    label_binarizer: A sklearn fitted label binarizer
+    threshold: Float between 0 and 1
+    probabilities: Whether Y_pred_proba will contain probabilities or just predictions
+
+    Returns:
+        A list of dictionaries for each prediction, e.g.
+        [{"tag": "Malaria", "Probability": 0.5}, ...]
```

Comment:
is that the format? isn't it? `[{"Malaria": 0.5, ...}]` also this is what's returned when probabilities are True otherwise its a list